internal/auth/client.go

```diff
@@ -209,3 +209,14 @@ func decodeError(err error) error {
 	}
 	return err
 }
+
+type identityRes struct {
+	id       string
```

Comment:
Change ID to `subject` or whatever it represents; just `id` is misleading since this is not the response ID.